cypress/support/index.js

```diff
@@ -1,7 +1,7 @@
-import { registerCommands } from '@dhis2/cypress-commands'
+import { enableAutoLogin } from '@dhis2/cypress-commands'
 
 import { enableNetworkShim, loginAndPersistSession } from './server'
 
-registerCommands()
+enableAutoLogin()
 enableNetworkShim()
 loginAndPersistSession()
```

Comment:
For DV I removed `loginAndPersistSession` completely, but maybe its needed here for testing stubs?

Reply:
You're right. Removed that file and some more code that is handled by enableAutoLogin.